spec/classes/init_spec.rb

```diff
@@ -33,6 +33,7 @@
         it "stores the configuration and identity on the node" do
           if os_family == 'Windows'
             expect(subject).to contain_registry_value('HKLM\Software\CyberArk\Conjur\ApplianceUrl')
+                                   .with("data" => 'https://conjur.test/api')
             expect(subject).to contain_registry_value('HKLM\Software\CyberArk\Conjur\SslCertificate')
             expect(subject).to contain_registry_value('HKLM\Software\CyberArk\Conjur\Account')
             expect(subject).to contain_registry_value('HKLM\Software\CyberArk\Conjur\Version')
```

Comment:
Can these other 3 assertions also have a '.with(`? Would be nice to assert that the value matches.

Reply:
Took this out, i'd mistakenly added it while experimenting